### PR TITLE
EVEREST-287 Drop DBaaS

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-0

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

**Review by BupycHuk — APPROVED** [review]

**Comment by BupycHuk** [comment]
@gen1us2k could you fix merge conflicts?

**Comment by gen1us2k** [comment]
Yep. On it and I'll review it by myself before asking it again

File: Makefile.include [hunk 1]
```diff
@@ -43,11 +39,11 @@ gen: clean            ## Generate files
 
 gen-api:            ## Generate PMM API
 	# generated by descriptors target
-	bin/buf breaking --against descriptor.bin api
+	#bin/buf breaking --against descriptor.bin api
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply Makefile.include:42]: yes, it is, and actually you can update descriptors by running `make descriptors` command and everything will be fine

File: update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml [hunk 2]
```diff
@@ -83,10 +83,6 @@
       ignore_errors: true
       # FIXME: we have a race condition here. We generate grafana supervisor config in pmm-managed and it doesn't exist at this stage
 
-    - name: Remove old DBaaS dashboard (Before provisioning times)
-      command: sqlite3 /srv/grafana/grafana.db -cmd ".timeout 60000" "DELETE FROM dashboard WHERE title = 'DBaaS' AND slug = 'dbaas';"
-      changed_when: true
-
```
Comment (BupycHuk) [inline update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml:89]: we still need it  to delete old dashboards

Comment (gen1us2k) [reply update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml:89]: I'll revert it before the merge. No worries.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -3,8 +3,7 @@
 /agent/runner/                          @artemgavrilov @percona/pmm-review-be
 /api/                                   @BupycHuk @percona/pmm-review-be
 /managed/services/checks/               @idoqo @percona/pmm-review-be
-/managed/services/dbaas                 @gen1us2k @recharte @percona/pmm-review-be
-/managed/services/management/dbaas      @gen1us2k @recharte @percona/pmm-review-be
+/managed/                               @percona/pmm-review-be
```
[the remark this answers is not in the document]
Comment (gen1us2k) [reply .github/CODEOWNERS:6]: Sure

**Comment by gen1us2k** [comment]
The only failed run
https://github.com/percona/pmm/actions/runs/6340498602/job/17222014123?pr=2510
and I assume that's not related to the changes

File: update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml (hunk 2 above)
Comment (gen1us2k) [reply update/ansible/playbook/tasks/roles/dashboards_upgrade/tasks/main.yml]: Done

File: Makefile.include (hunk 1 above)
[the remark this answers is not in the document]
Comment (gen1us2k) [reply Makefile.include]: Nice.

**Review by atymchuk — APPROVED** [review]

**Pull request merged**